testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/ModuleTestCase.java

```diff
@@ -186,6 +186,20 @@ public void addRemoveModuleMetaInf() throws Exception {
         testModuleAndSlotExist(false, slot, getModulePath());
     }
 
+    @Test
+    public void addModuleWithWhitespacesInDependencies() throws Exception {
+        // should trim the surrounding whitespaces
+        try {
+            cli.sendLine("module add --name=" + MODULE_NAME
+                    + " --resources=" + jarFile.getAbsolutePath()
+                    + " --dependencies=[ org.jboss.logging ]"
```

Comment:
I just checked, '[' and ']' are supported and needed to escape ',' when command is passed to CLI process:
jboss-cli.sh "module add --name=mod --resources=my.jar --dependencies=[ibm.jdk,javax.org]"

If not used, the ',' is interpreted as a command separator. So the '[' and ']' are only there to escape ','.

I don't think that we need them in this particular case. We could keep them but this is a bit strange to escape ',' in a context where we don't have to.